src/test/kotlin/hwr/oop/projects/peakpoker/core/game/GameTest.kt

```diff
@@ -96,6 +96,203 @@ class GameTest : AnnotationSpec() {
+        val testGame = Game(
+            1032, 10, 20,
+            listOf(Player("Hans"), Player("Peter"), Player("Max"))
+        )
+
+        val initialPlayer = testGame.getCurrentPlayer()
+        testGame.makeTurn()
+        val nextPlayer = testGame.getCurrentPlayer()
+
+        assertThat(nextPlayer.name).isNotEqualTo(initialPlayer.name)
+        assertThat(testGame.currentPlayerIndex).isEqualTo(0)
+    }
+
+    @Test
+    fun `checkPlayerValidity returns false for existing players and true for new players`() {
+        val testGame = Game(
+            1033, 10, 20,
+            listOf(Player("Hans"), Player("Peter"), Player("Max"))
+        )
+
+        val existingPlayer = Player("Hans")
+        val newPlayer = Player("Sara")
+
+        assertThat(testGame.checkPlayerValidity(existingPlayer)).isFalse()
+        assertThat(testGame.checkPlayerValidity(newPlayer)).isTrue()
+    }
+
+    @Test
+    fun `getHighestBet handles edge cases with equal bets`() {
+        val player1 = Player("Hans")
+        val player2 = Player("Peter")
+        val player3 = Player("Max")
+        val testGame = Game(1040, 10, 20, listOf(player1, player2, player3))
+
+        player1.raiseBet(30)
+        player2.raiseBet(30)
+        player3.raiseBet(30)
+
+        assertThat(testGame.getHighestBet()).isEqualTo(30)
+    }
+
+    @Test
+    fun `getHighestBet works with zero bets`() {
+        val player1 = Player("Hans")
+        val player2 = Player("Peter")
+        val player3 = Player("Max")
+        val testGame = Game(1041, 10, 20, listOf(player1, player2, player3))
+
+        assertThat(testGame.getHighestBet()).isEqualTo(0)
+    }
+
+    @Test
+    fun `getHighestBet returns correct value when some bets are the same`() {
+        val player1 = Player("Hans")
+        val player2 = Player("Peter")
+        val player3 = Player("Max")
+        val testGame = Game(1042, 10, 20, listOf(player1, player2, player3))
+
+        player1.raiseBet(10)
+        player2.raiseBet(30)
+        player3.raiseBet(10)
+
+        assertThat(testGame.getHighestBet()).isEqualTo(30)
+    }
+
+    @Test
+    fun `big blind amount can be equal to small blind amount`() {
+        val game = Game(
+            1050, 20, 20,
+            listOf(Player("Hans"), Player("Peter"), Player("Max"))
+        )
+
+        assertThat(game.smallBlindAmount).isEqualTo(20)
+        assertThat(game.bigBlindAmount).isEqualTo(20)
+    }
+
+    @Test
+    fun `getSmallBlindIndex returns correct value`() {
+        val testGame = Game(
+            1060, 10, 20,
+            listOf(Player("Hans"), Player("Peter"), Player("Max"))
+        )
+
+        assertThat(testGame.smallBlindIndex).isEqualTo(0)
+    }
+
+    @Test
+    fun `getPot returns correct value`() {
+        val testGame = Game(
+            1061, 10, 20,
+            listOf(Player("Hans"), Player("Peter"), Player("Max"))
+        )
+
+        testGame.pot = 100
+
+        assertThat(testGame.pot).isEqualTo(100)
+    }
```

Comment:
No, this is fatal: This is not the way the pot should be returned and i imagined the functionality. You must call the `calculatePot()` function on the game. So:
```suggestion
    @Test
    fun `getPot returns correct value`() {
        val testGame = Game(
            1061, 10, 20,
            listOf(Player("Hans"), Player("Peter"), Player("Max"))
        )

        testGame.calculatePot() = 100

        assertThat(testGame.pot).isEqualTo(100)
    }
```